pyproject.toml

```diff
@@ -21,7 +22,8 @@ secureli_init = "secureli init -y"
 install = "poetry install"
 lint = "black --check ."
 precommit = "pre-commit run -a"
-test = ["init", "lint", "coverage_run"]
+test = ["init", "lint", "coverage_run", "coverage_html", "bats"]
```

Comment:
if you use coverage_report instead of just _run, it will produce a text report at the command line. I honestly don't know why you want html because that either makes a file no one looks at or it pops up a browser window (which I find annoying at least).

```[chore/53_Add_BATS_Testing][~/secureproject/secureli]$ poetry run poe coverage_report
Poe => coverage report
Name                                       Stmts   Miss Branch BrPart  Cover
----------------------------------------------------------------------------
secureli/abstractions/echo.py                 44      0      8      0   100%
secureli/abstractions/lexer_guesser.py         8      0      0      0   100%
secureli/abstractions/pre_commit.py          275      0    110      0   100%
secureli/actions/action.py                   113      0     24      0   100%
secureli/actions/build.py                     10      0      0      0   100%
secureli/actions/initializer.py               13      0      2      0   100%
secureli/actions/scan.py                     102      0     40      0   100%
secureli/actions/setup.py                     10      0      0      0   100%
secureli/actions/update.py                    31      0      6      0   100%
secureli/container.py                         45      0      0      0   100%
secureli/main.py                              30      0      0      0   100%
secureli/repositories/repo_files.py           35      0     12      0   100%
secureli/repositories/secureli_config.py      25      0      2      0   100%
secureli/repositories/settings.py             51      0      6      1    98%
secureli/resources/read_resource.py            8      0      2      0   100%
secureli/resources/slugify.py                 11      0      2      0   100%
secureli/services/git_ignore.py               45      0     10      0   100%
secureli/services/language_analyzer.py        33      0      8      0   100%
secureli/services/language_support.py         18      0      0      0   100%
secureli/services/logging.py                  59      0      0      0   100%
secureli/services/scanner.py                  84      0     24      0   100%
secureli/services/secureli_ignore.py          11      0      4      0   100%
secureli/services/updater.py                  35      0     10      0   100%
secureli/settings.py                          23      0      2      0   100%
secureli/utilities/git_meta.py                20      0      4      0   100%
secureli/utilities/patterns.py                12      0      6      0   100%
secureli/utilities/secureli_meta.py            3      0      0      0   100%
secureli/utilities/usage_stats.py              9      0      2      0   100%
----------------------------------------------------------------------------
TOTAL                                       1163      0    284      1    99%
```

Reply:
I used coverage_html as it quietly generates an HTML report that you can look at to see exactly which lines weren't covered which is details not given by coverage_report.  Also, since it quietly generates the report in the background and doesn't seem to take any longer, I felt it was a nice little nugget that saves the developers an extra step.  I can take it out if you think it doesn't belong.

Reply:
That makes sense. It would be nice if there was a way to combine them because it's nice to have the text report right away on the command line and in the pipeline.